api/v1beta1/queue_types.go

```diff
@@ -21,18 +21,19 @@ import (
 
 // QueueSpec defines the desired state of Queue
 type QueueSpec struct {
-	// Name of the queue; required property
+	// Name of the queue; required property.
 	// +kubebuilder:validation:Required
 	Name string `json:"name"`
 	// Default to vhost '/'
 	// +kubebuilder:default:=/
 	Vhost string `json:"vhost,omitempty"`
 	Type  string `json:"type,omitempty"`
-	// When set to false queues does not survive server restart
+	// When set to false queues does not survive server restart.
 	Durable bool `json:"durable,omitempty"`
-	// when set to true, queues that has at least one consumer before, are deleted after last consumer unsubscribes
+	// when set to true, queues that has at least one consumer before, are deleted after last consumer unsubscribes.
```

Comment:
// when set to true, queues that have had at least one consumer before are deleted after the last consumer unsubscribes.